index.html

```diff
@@ -23,8 +23,7 @@
   </head>
```

Comment:
- [ ] You have done a good job with most of the accessibility issues :+1: but I would suggest using a more descriptive title because "Personal Portfolio" could be too general, you could use e.g., "Your Name's Portfolio" or "Your Name - Full Stack Developer" :wink: